setup.py

```diff
@@ -188,7 +191,8 @@ def get_data_files(name, version, fullname):  # pylint: disable=R0912
         set_conf_files(data_files, src=["config/openbsd/waagent.conf"])
         set_openbsd_rc_files(data_files)
     elif name == 'debian':
-        set_bin_files(data_files, dest=agent_bin_path)
+        set_bin_files(data_files, dest=agent_bin_path,
+                      src=["bin/py3/waagent", "bin/waagent2.0"])
```

Comment:
same for debian, we should probably leave it alone... or was this needed?

Reply:
same like ubuntu.. all debian distros using py3 packer template

Reply:
k